apps/core/components/ItineraryCard/RenderTripSectorItem.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+import * as React from 'react';
+import { graphql, createFragmentContainer } from '@kiwicom/margarita-relay';
+
+import StopoverDuration from './StopoverDuration';
+import TripSector from './TripSector';
+import ItineraryCardRow from './ItineraryCardRow';
+import type { RenderTripSectorItem as RenderTripSectorItemType } from './__generated__/RenderTripSectorItem.graphql';
+
+type Props = {|
+  +data: ?RenderTripSectorItemType,
+|};
+
+function RenderTripSectorItem({ data, nextSectorDepartureUtc }: Props) {
```

Comment:
`nextSectorDepartureUtc` is no longer a prop here is it?

Reply:
Removed